sites/kit.svelte.dev/src/routes/home/Showcase.svelte

```diff
@@ -0,0 +1,106 @@
+<script>
+	import Section from '@sveltejs/site-kit/components/Section.svelte';
+	import pudding from './showcase/pudding.jpg';
+	import evidence from './showcase/evidence.jpg';
+	import fieuzal from './showcase/fieuzal.jpg';
+	import gitbook from './showcase/gitbook.jpg';
+	import monogram from './showcase/monogram.jpg';
+	import raster from './showcase/raster.jpg';
+	import trading_strategy from './showcase/trading-strategy.jpg';
+	import senja from './showcase/senja.jpg';
+
+	const showcase = [
+		{ url: 'pudding.cool', image: pudding },
+		{ url: 'fieuzal.com', image: fieuzal },
+		{ url: 'evidence.dev', image: evidence },
+		{ url: 'gitbook.com', image: gitbook },
+		{ url: 'raster.app', image: raster },
+		{ url: 'monogram.io', image: monogram },
+		{ url: 'tradingstrategy.ai', image: trading_strategy },
+		{ url: 'senja.io', image: senja }
+	];
+</script>
+
+<Section --background="var(--background-2)">
+	<h2>showcase</h2>
+
+	<div class="showcase">
+		{#each showcase as { url, image }}
+			<a class="drop-shadow" href="https://{url}" target="_blank" rel="noreferrer">
+				<img src={image} alt="Screenshot of {url}" />
+				<span>{url}</span>
```

Comment:
Having both of these is giving the link the accessible name of "Screenshot of evidence.dev evidence.dev". I *think* I'd give the img an empty alt instead since it's decorative and then the link will just be named "evidence.dev"

```suggestion
				<img src={image} alt="" />
				<span>{url}</span>
```